### DISCUSSION TITLE
REQ: Process Qizmo voice packets (low priority)

### DISCUSSION THREAD

**Discussion by meag** [ezQuake/ezquake-source#499] [Ideas]
**Describe the solution you'd like**
Some old demos have svc_qizmovoice (83) packets, that the user should be able to play back when watching the demo.

Currently these are read & skipped, although some-one seems to have worked out some pattern to the packet in the past.  (https://github.com/ezQuake/ezquake-source/blob/master/cl_parse.c#L3456-L3466)

No idea if this was ever solved further.

**Describe alternatives you've considered**
Watch the demos through qizmo with game sounds disabled and commentary turned on.  Record audio with external tool, then re-encode the demo with svc_qizmovoice removed and svc_fte_voicechat packets inserted instead.

**Comment by HoytHernandez** [comment]
Das ist ein wirklich interessanter Wunsch. Die alten Qizmo-Sprachpakete sind Teil der Quake-Geschichte, und es wäre großartig, wenn Demos sie einfach nativ abspielen könnten, anstatt sie zu überspringen. Selbst eine teilweise Dekodierung basierend auf dem bestehenden Muster wäre ein guter Anfang, besonders wenn sie in gängigen Fällen funktioniert. Die Umgehungslösung mit der Aufnahme über Qizmo wirkt umständlich und fehleranfällig, daher wäre eine direkte Unterstützung innerhalb der Engine definitiv eleganter. Wildz bietet modernes Gaming https://wildzcasino-de.com/ mit spannender Spielauswahl. Selbst als Feature mit niedriger Priorität würde es einen großen Mehrwert für den Erhalt klassischer Demo-Inhalte bieten.

**Comment by JesseReed123** [comment]
It would be great if old demos containing svc_qizmovoice packets could be properly played back within the client. Currently, these packets are just read and skipped, even though there’s some understanding of their pattern in the source code. Right now, the only workaround is to watch the demo in Qizmo with game sounds off and commentary on, record the https://roobetcom.pro/ audio externally, then re-encode the demo by replacing svc_qizmovoice with svc_fte_voicechat. A built-in solution to decode and play these packets would make demo review much smoother.
